test/e2e_node/system/specs/gke.yaml

```diff
@@ -167,10 +167,11 @@ kernelSpec:
     description: 'Enabled the SECCOMP application API.'
   - name: SECURITY_APPARMOR
     description: 'Enable for AppArmor support.'
-  - name: CC_STACKPROTECTOR_STRONG
+  - name: CC_STACKPROTECTOR_STRONG # until 4.17
```

Comment:
Would you mind making it clear in your comments that the version numbers here refer to linux kernel versions?

Reply:
That is a good idea. Thanks!